### DISCUSSION TITLE
自定义ui指令里，如何用svg图标

### DISCUSSION THREAD

**Discussion by 13701138894** [dream-num/univer#3663] [Q&A | 常见问题]
//     this.disposeWithMe(componentManager.register('fx', SlideSingle))
//     this.disposeWithMe(componentManager.register('px', FilteredSortDescSingle))
//     this.disposeWithMe(componentManager.register('report', DocSingle))

用js，不用jsx和tsx，不用框架，不用第三方库，想使用svg，该如何操作才能自定义图标

**Comment by 13701138894** [comment]
可以了

**Reply by Astronaut224** [reply]
请问自定义ui在vue2里用js可以实现吗，能参考一下是怎么写的吗
